src/RA_Core.cpp

```diff
@@ -1182,6 +1190,9 @@ void RememberWindowSize(HWND hDlg, const char* sDlgKey)
     iter->second.nHeight = rc.bottom - rc.top;
 }
 
+_Use_decl_annotations_
+BOOL CALLBACK SetDirectory() { return SetCurrentDirectory(NativeStr(g_sHomeDir).c_str()); }
+
```

Comment:
Found this to be extremely repetitive so I made a function for it.

Reply:
but you never call the function. 

also, recommend a more descriptive name. maybe `ChangeToHomeDirectory()`?

Reply:
Yeah I think that's a better name, since it always sets it to the home directory.